src/main/java/com/quorum/gauge/common/QuorumNetworkProperty.java

```diff
@@ -108,6 +108,8 @@ public void setPort(int port) {
         private String validatorAddress;
         @JsonProperty("enode-address")
         private String enode;
+        @JsonProperty("graphql-url")
```

Comment:
This is deprecated. Please remove

Reply:
Maybe we should consider removing the multiple existing @JsonProperty as well?